index.js

```diff
@@ -62,8 +63,9 @@ NDRandom.prototype.getValue = function (udr) {
           apa = (udr * this.ndc[this.ndc.length - 1].cumArea) - an1, // the target partial volume of the trapezoid
           a = m/2, // first coefficient of the quadratic equation
           b = yn1-(xn1 * m), // second coefficient of the quadratic equation
-          c = ((xn1 * xn1 * m/2) - (xn1 * yn1) - apa), // contant value of the quadratic equation
-          aos = (-1 * b) / (2 * a), // axis of symmetry
+          c = ((xn1 * xn1 * m/2) - (xn1 * yn1) - apa); // contant value of the quadratic equation
+		    if(a===0) return undefined; // prevent divide by zero in zero areas
```

Comment:
Interesting. I'd never considered distributions with zero areas but I guess they could be useful.

Reply:
it is useful. I sample the actual distribution with a histogram and I want to recreate it.